scss/components/_activity-card.scss

```diff
@@ -169,11 +169,12 @@
 .civicase__activity-card-row {
   align-items: flex-start;
   display: flex;
-  line-height: 1.8em;
+  line-height: 18px;
```

Comment:
Instead of all this fix, if you do
```css
.civicase__contact-icon .material-icons {
  line-height: inherit;
}
```
seems to fix the issue to. Could you check.